src/abbreviations.jl

```diff
@@ -445,6 +445,58 @@ function format(::TypeDefinition, buf, doc)
     end
 end
 
+"""
+The singleton type for [`README`](@ref) abbreviations.
+"""
+struct Readme <: Abbreviation end
+"""
+    README
+
+An [`Abbreviation`](@ref) for including the package README.md.
+
+!!! note
+    The README.md file is interpreted using Julias markdown parser, which
```

Comment:
`interpreted as "Julia flavored Markdown"`, with link to: https://docs.julialang.org/en/v1/manual/documentation/#Markdown-syntax-1 ?